pkg/network/tracer/ebpf_conntracker.go

```diff
@@ -57,6 +59,13 @@ type ebpfConntracker struct {
 
 // NewEBPFConntracker creates a netlink.Conntracker that monitor conntrack NAT entries via eBPF
 func NewEBPFConntracker(cfg *config.Config) (netlink.Conntracker, error) {
+	ksymPath := filepath.Join(cfg.ProcRoot, "kallsyms")
+	missing, err := ddebpf.VerifyKernelFuncs(ksymPath, []string{"__nf_conntrack_hash_insert"})
```

Comment:
Does the netlink-based conntracker work if the kernel module is not loaded? Perhaps this should be a level higher and apply to both netlink and eBPF conntrackers?